### DISCUSSION TITLE
ansible-playbook command not found

### DISCUSSION THREAD

**Discussion by ebhaskarreddy** [learndevopsonline/learndevopsonline#1581] [Q&A] [answered]
how it resolve can anyone help me 
![image](https://github.com/learndevopsonline/learndevopsonline/assets/149461818/3ed7abb6-ad19-4720-9d54-afa6a976defd)

**Comment by sreedharm07** [comment, marked as answer]
Hi @ebhaskarreddy 
It is supposed to be **playbook** please correct it and try.
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/fc300030-74ea-4972-a1ca-e62797a58fa1)

**Reply by ebhaskarreddy** [reply]
tq

**Reply by ebhaskarreddy** [reply]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/149461818/b6e843cf-e3d7-43ab-ae4d-789ab0d049c9)

**Reply by sreedharm07** [reply]
please change this as per documentation. there you should download frontend code.
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/aca7386b-cfa9-4572-b862-ca50306d79d1)

**Reply by ebhaskarreddy** [reply]
changed it  and now got like this 
![image](https://github.com/learndevopsonline/learndevopsonline/assets/149461818/f89c10a8-0edc-401c-be27-c15aed1a7190)

**Reply by ebhaskarreddy** [reply]
remote host means domain  , am i right

**Reply by sreedharm07** [reply]
@ebhaskarreddy 
**rdevops650nline.online** still your dns was not available in all locations. thats why its not able to start.

**Reply by ebhaskarreddy** [reply]
how you verfied dns not available in the above one

**Reply by sreedharm07** [reply]
dns checker

**Reply by ebhaskarreddy** [reply]
am asking in the above image where you find dns is not available , through dns checker is fine

**Reply by sreedharm07** [reply]
from your github in expence.conf file
